src/fastoad/model_base/flight_point.py

```diff
@@ -83,6 +83,7 @@ class FlightPoint:
     isa_offset: float = 0.0  #: temperature deviation from Standard Atmosphere
     ground_distance: float = 0.0  #: Covered ground distance in meters.
     mass: float = None  #: Mass in kg.
+    consumed_fuel: float = 0.0  #: Consumed fuel since mission start, in kg.
```

Comment:
Er... that's precisely the point of the PR 😅 
Maybe you missed we have a dataclass here?

Reply:
From what I can read this PR adds the fuel consumed since the beginning of the mission,
 `consumed_fuel: float = 0.0  #: Consumed fuel since mission start, in kg.`
 I was suggesting the fuel consumed at each point of the flight (instantaneous fuel consumption * time_step = sfc * thrust* time_step)

Reply:
Ok, my bad. Yet I am not convinced it would be a good thing.
Do you have in mind a use case where it would be useful to have such data during the computation?
As you know, all available data in FlightPoint are state variables, with a link to physical properties. Here, you talk about a value that is entirely linked to the chosen time step, which is rather different.

Reply:
For the fuel consumed at each time step, not really. 
However, I suggest that we add what I called the instantaneous fuel consumption in my previous comment. It is not linked to the time step (😉 ) and, for GA aircraft, you can often see the fuel consumption in kg/h (actually gal/h but I doubt it'll be possible to do it. This can serve as a very good comparaison point as opposed to the sfc alone 😄

Reply:
You'll find me annoying, but again, I'm not fond of the idea. I understand the need, but thrust and SFC are already available for each flight point, and it is very easy to get the instantaneous fuel consumption from them.
TBH, I am already annoyed to add the total consumed fuel because it can be obtained from initial and current mass, but I came to that because it cannot be computed when you have only the data of one flight point.
Here we talk about pure "convenience fields", and if we get into that, we could probably add a lot more... and I'm not willing to do that 😅 

I would prefer thinking about proposing a way for user to specify custom additional fields, derived from existing fields with some arithmetic rules.

Reply:
Got it !

I'll let you merge this PR but cant wait to discuss your last idea